locale/ko/docs/guides/index.md

```diff
@@ -1,6 +1,26 @@
 ---
 title: 가이드
-layout: guides-index.hbs
+layout: docs.hbs
 ---
 
 # 가이드
+
+## General
```

Comment:
Probably best to leave this change up to the localization teams so they can pick language-appropriate titles.

Reply:
/cc @nodejs/nodejs-ko